src/_guides/language/language-tour.md

```diff
@@ -1742,9 +1751,9 @@ allows you to write more fluid code.
 
 Consider the following code:
 
-<?code-excerpt "language-tour/cascade-operator/web/main.dart"?>
+<?code-excerpt "test/language_tour/browser_test.dart (cascade-operator)"?>
 {% prettify dart %}
-querySelector('#button') // Get an object.
+querySelector('button') // Get an object.
```

Comment:
Why `button` instead of `#button`? If it's because it seems like a bad name, how about something like `#confirm_button`?

Reply:
If there is a single button on the page, using `button` as a selector is just fine. Using `#confirm_button` would be better, though longer, and it doesn't add anything to the point being made in the prose (IMHO).

Reply:
While what Patrice says is true, this does rub me the wrong way. I'd consider it a potential error (or source of later error) if I saw this. 

How about `querySelector('#confirm')` if you want to stay terse? In my opinion, `#button` is also fine, as is `#confirm_button`.

Reply:
Ok, I'll switch to `#confirm` instead of `button` as selector.